pyiron_base/interfaces/has_hdf.py

```diff
@@ -106,6 +106,35 @@ class HasHDF(ABC):
     >>> hdf.remove_file()
     >>> pr.remove(enable=True)
 
+    When using this class as a mixin that also derive from classes that have a
+    legacy implementation here's a simple recipe
+
+    >>> class MyOldClass:
+    ...     def to_hdf(self, hdf, group_name):
+    ...         pass # snip
+    ...     def from_hdf(self, hdf, group_name):
+    ...         pass # snip
+    >>> class MyDerivedClass(MyOldClass, HasHDF):
+    ...     def to_hdf(self, hdf, group_name):
+    ...         MyOldClass.to_hdf(self, hdf=hdf, group_name=group_name)
+    ...         HasHDF.to_hdf(self, hdf=hdf, group_name=group_name)
+    ...     def from_hdf(self, hdf, group_name):
+    ...         pass # snip
+
+    i.e. explicitly call both methods with the same group_name.  The call to
+    :meth:`.HasHDF.to_hdf` has to be last so that the type information is
```

Comment:
```suggestion
    :meth:`HasHDF.to_hdf` has to be last so that the type information is
```

I think? If that extra dot was intentional then ignore this suggestion and just explain to me what I'm missing ;)

Reply:
Normally you have to give the full module path to an object so that sphinx does the referencing correctly.  With the dot you can skip part of it.